### ISSUE TITLE
alt text on claim btn (when disabled) should become a tooltip

### ISSUE THREAD

**Issue by karelstriegel** [FundRequest/platform#326]
### Expected behaviour
When hovering over the claim butten (is disable state) on a request detail page, the alt text should be replace by a tooltip, in the same style as the tooltip we see in the wizard steps to fund a request.

**Comment by Simsteve7** [comment]
<img width="616" alt="screen shot 2018-05-30 at 15 16 31" src="https://user-images.githubusercontent.com/1853096/40722392-7dab2fce-641c-11e8-9af4-a5ab3b8da9ed.png">
